master/internal/core_experiment.go

```diff
@@ -25,6 +25,7 @@ import (
 	expauth "github.com/determined-ai/determined/master/internal/experiment"
 	"github.com/determined-ai/determined/master/internal/project"
 	pkgTemplate "github.com/determined-ai/determined/master/internal/template"
+	u "github.com/determined-ai/determined/master/internal/user"
```

Comment:
imo, don't rename this import, rename the local var user that is conflicting (maybe `owner` is a good name for it)